### PR TITLE
New App: Powerdns Recursor

### PR DESCRIPTION
**Pull request by murrant**
Polling must have access to stats for the device, port 8082 (Agent file could be added to avoid that)

**Graphs:**
Questions
Answers
Cache Performance
Cache Size
Outbound Queries

**Graph changes**
generic_multi:print_total added, allow per-dataset colors, document variables
generic_multi_line: allow per-dataset colors, document variables

### CODE REVIEW

File: doc/mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+site_name: LibreNMS
```
Comment (laf) [inline doc/mkdocs.yml]: Looks like this slipped in by mistake.

**Comment by laf** [comment]
Is the Doc worth having, we already have Agent-Setup doc, maybe it should be cleaned up and merged into one.

Seems like this branch also has the IBM stuff in.

**Comment by murrant** [comment]
Yeah,  I think you are right on the docs.  I'm not sure how that ibm-amm snuck in :/

**Comment by murrant** [comment]
I might send a PR with some changes to the agent-setup.md separately.

**Comment by vitalisator** [comment]
the agent script collects data as expected, but the poller process do not grab any data.

running poller in debug mode: http://pastebin.com/N6z1Z9ci
....
 powerdns-recrusor  //typo
....

**Comment by murrant** [comment]
@vitalisator If you want to collect data from the agent, you need to run `./poller.php -h kx-dns-01.cmn.local -m unix-agent,applications -d`

**Comment by vitalisator** [comment]
ok, new try: http://pastebin.com/eB1YXUha
it seems that the agent data is grabbed but my graphs are empty

**Pull request merged**